pom.xml

```diff
@@ -824,13 +819,7 @@
         <dependency>
             <groupId>org.onebusaway</groupId>
             <artifactId>onebusaway-gtfs</artifactId>
-            <version>1.4.17</version>
-            <exclusions>
-                <exclusion>
-                    <groupId>org.slf4j</groupId>
-                    <artifactId>slf4j-simple</artifactId>
-                </exclusion>
-            </exclusions>
```

Comment:
Why was this exclusion removed?

Reply:
Because we should never have needed it in the first place. This artifact included an slf4j implementation in its dependencies which competed with our own (log back). I removed it upstream.